engine/src/main/java/org/terasology/particles/updating/ParticleUpdaterImpl.java

```diff
@@ -299,29 +302,39 @@ private void updateEmitter(final ParticleEmitterComponent particleEmitter, final
         }
     }
 
-    private void updateParticleSystem(final ParticleEmitterComponent partSys, final float delta) {
-        if (partSys.enabled && (partSys.particleSpawnsLeft == ParticleEmitterComponent.INFINITE_PARTICLE_SPAWNS || partSys.particleSpawnsLeft > 0)) {
-            updateEmitter(partSys, 0, delta); // Emit particles
+    private void updateParticleEmitters(final ParticleEmitterComponent emitter, final float delta) {
```

Comment:
Would be nice to see some Javadoc here for future contributors (and people not familiar with all the details, just like me ^^)